core/systems/crew/context.ex

```diff
@@ -104,6 +97,31 @@ defmodule Systems.Crew.Context do
     )
   end
 
+  def count_started_tasks(crew) do
+    from(t in Crew.TaskModel,
+      where:
+        t.crew_id == ^crew.id and
+        t.status == :pending and
+        t.expired == false and
+        not is_nil(t.started_at),
+      select: count(t.id)
+    )
+    |> Repo.one()
+
+  end
+
+  def count_applied_tasks(crew) do
+    from(t in Crew.TaskModel,
+      where:
+        t.crew_id == ^crew.id and
```

Comment:
Je zou een deel van de code in een losse functie kunnen plaatsen en dan de query uitbreiden met enkel de `not is_nil` etc.

Reply:
ja idd, zoals: `defp applied_tasks_query` do ..